src/scripts/main.js

```diff
@@ -1,3 +1,306 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tBody = table.querySelector('tbody');
+const sortedCols = {};
+let cellEditing = null;
+let cellInput = null;
+let initialCellText = '';
+let selectedRow;
+
+const getSalary = (salary) => {
+  return salary.innerText.slice(1).split(',').join('');
+};
+
+const sortByText = (rowsArray, index, inverseSort) => {
+  if (inverseSort === true) {
```

Comment:
If statement convert this value to the boolean by default
```suggestion
  if (inverseSort) {
```